doc/release_notes/release_notes_0.5.rst

```diff
@@ -56,8 +56,7 @@ Slim Bootloader is a modern, flexible, light-weight, open source
 reference boot loader with key benefits such as being fast, small,
 customizable, and secure. An end-to-end reference build with
 ACRN hypervisor, Clear Linux OS as SOS, and Clear Linux OS as UOS has been
-verified on UP2/SBL board. See the :ref:`using-sbl-up2` documentation
-for step-by-step instructions.
+verified on UP2/SBL board.
```

Comment:
FYI, I removed references in the release notes to docs that were deleted in the 2.2 release (but left the titles when appropriate).  The original release notes for each release remain as they were since the documents still exist in that tagged release.